src/backend/__tests__/utils.test.ts

```diff
@@ -257,59 +247,4 @@ describe('backend/utils.ts', () => {
       expect(utils.bytesToSize(2059 * 1024 * 3045 * 4000)).toEqual('23.36 TB')
     })
   })
-
-  describe('extractZip', () => {
-    let testCopyZipPath: string
-    let destFilePath: string
-
-    beforeEach(() => {
-      const testZipPath = path.resolve('./src/backend/__mocks__/test.zip')
-      //copy zip because extract will delete it
-      testCopyZipPath = path.resolve('./src/backend/__mocks__/test2.zip')
-      copyFileSync(testZipPath, testCopyZipPath)
-      destFilePath = path.resolve('./src/backend/__mocks__/test')
-    })
-
-    afterEach(async () => {
-      const extractPromise = utils.extractZip(testCopyZipPath, destFilePath)
-      await extractPromise
-      expect(extractPromise).resolves
-
-      const testTxtFilePath = path.resolve(destFilePath, './test.txt')
-      console.log('checking dest file path ', testTxtFilePath)
-      expect(existsSync(testTxtFilePath)).toBe(true)
-
-      const testMessage = readFileSync(testTxtFilePath).toString()
-      console.log('unzipped file contents: ', testMessage)
-      expect(testMessage).toEqual('this is a test message')
-
-      //extract deletes the zip file used to extract async so we wait and then check
-      await utils.wait(100)
-      expect(existsSync(testCopyZipPath)).toBe(false)
-
-      //clean up test
-      rmSync(testTxtFilePath)
-      rmdirSync(destFilePath)
-      expect(existsSync(testTxtFilePath)).toBe(false)
-      expect(existsSync(destFilePath)).toBe(false)
-    })
-
-    test('extract a normal test zip', async () => {
-      console.log('extracting test.zip')
-    })
-
-    test('extract a test zip with non ascii characters', async () => {
-      const renamedZipFilePath = path.resolve(
-        './src/backend/__mocks__/谷���新道ひばりヶ�.zip'
-      )
-      renameSync(testCopyZipPath, renamedZipFilePath)
-      testCopyZipPath = renamedZipFilePath
-    })
```

Comment:
This is an important test case as we have had games with non ascii characters fail to extract before. I do not see it in `ExtractZipService.test.ts`. Can you add it?

Reply:
Done